components/bl-double-calendar-component/src/lib/classnames.js

```diff
@@ -0,0 +1,48 @@
+/*!
```

Comment:
I assume, that must be builded through webpack into one minified file.

@Valodya what do you think?